numpy/_core/fromnumeric.py

```diff
@@ -2953,12 +2953,12 @@ def cumsum(a, axis=None, dtype=None, out=None):
     return _wrapfunc(a, 'cumsum', axis=axis, dtype=dtype, out=out)
 
 
-def _ptp_dispatcher(a, axis=None, out=None, keepdims=None):
+def _ptp_dispatcher(a, axis=None, out=None, keepdims=None, where=None):
     return (a, out)
 
 
 @array_function_dispatch(_ptp_dispatcher)
-def ptp(a, axis=None, out=None, keepdims=np._NoValue):
+def ptp(a, axis=None, out=None, keepdims=np._NoValue, where=np._NoValue):
```

Comment:
This should be kwarg only.

FWIW, I don't care much to do this, but I agree with Matti in that I am curious where this type of issues come from.  It feels like a lot more recently and context matters.

For example, if an automatic tool says: oh this might match better if it has a `where=`, then we'll never see the end of it.
OTOH, if you have an *actual* use, that is a reason to just do it (when there is little downside otherwise).

(The matter would be different if a maintainer spearheaded this so that we have dedicated review bandwidth.)